src/main/java/org/verdictdb/coordinator/QueryResultAccuracyEstimatorFromDifference.java

```diff
@@ -233,4 +215,18 @@ private boolean checkConverge() {
 
     return isValueConverged;
   }
+
+  public boolean checkIfQueryCountOnly() {
+    for (SelectItem sel:this.originalQuery.getSelectList()) {
+      if (sel instanceof AliasedColumn) {
```

Comment:
Instead of this, let's use the function similar to https://github.com/mozafari/verdictdb/blob/master/src/main/java/org/verdictdb/core/sqlobject/ColumnOp.java#L419

Please add a function called "isCountDistinctAggregate()" and call it here.